.buildkite/pipeline.yml

```diff
@@ -168,3 +168,28 @@ steps:
     plugins:
       - junit-annotate#v2.4.1:
           artifacts: build/TEST-go-integration*.xml
+
+  - group: "K8s tests"
+    key: "k8s-tests"
+    steps:
+      - label: "K8s tests: {{matrix.k8s_version}}"
+        env:
+          K8S_VERSION: "v{{matrix.k8s_version}}"
+          KIND_VERSION: "v0.20.0"
+        command: ".buildkite/scripts/steps/k8s-tests.sh"
+        artifact_paths:
+          - "build/TEST-**"
```

Comment:
I wonder what type of reports this generates, do we get JUnit reports?
If so it would be nice to incorporate the following plugin as well.
```
plugins:
      - junit-annotate#v2.4.1:
          artifacts: build/TEST-*.xml
```

So that in case of failure, we get the nice BK annotation and it would save some time to figure out what failed.